### PR TITLE
Typing AVDTP

### PR DESCRIPTION
**Pull request by zxzxwu**
Still, many commands wrappers are left untyped. Maybe in another PR we can replace `send_command()` with an overloaded form, which mapping command to their responses.

### CODE REVIEW

File: bumble/avdtp.py [hunk 1]
```diff
@@ -385,32 +408,37 @@ async def stop(self):
 
 
 # -----------------------------------------------------------------------------
-class MessageAssembler:  # pylint: disable=attribute-defined-outside-init
-    def __init__(self, callback):
-        self.callback = callback
```
Comment (barbibulle) [inline bumble/avdtp.py:390]: As a general design pattern, I'm not a big fan of using event emitters for direct 1:1 relationships like this, where there's normally just one client that wants to receive the data. Having a direct function allows better typing that generic events. Events are better suited for 1:M relationships, where you can dynamically add/remove listeners.

Comment (zxzxwu) [reply bumble/avdtp.py:390]: Done

File: bumble/avdtp.py [hunk 2]
```diff
@@ -551,7 +586,9 @@ def __str__(self):
 
 # -----------------------------------------------------------------------------
 class MediaCodecCapabilities(ServiceCapabilities):
-    def init_from_bytes(self):
+    media_codec_information: Union[bytes, SupportsBytes]
+
```
Comment (barbibulle) [inline bumble/avdtp.py:590]: missing `media_type` and `media_codec_type`?

**Review by barbibulle — APPROVED** [review]

File: bumble/avdtp.py (hunk 2 above)
Comment (zxzxwu) [reply bumble/avdtp.py:592]: They can be inferred from `__init__`, but I added hints still.

**Pull request merged**